### PR TITLE
Members-only authentication

### PR DESCRIPTION
**Pull request by mke2111**
Members-only:

This project is based on authenticating users on a given web page using devise gem in rails application setup.

A signed-in user can see who the author of a secret is unlike the unsigned-in user who is only authorized the view privilege.

Tailwindcss was used to incorporate styles to the various pages.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,67 @@
-# README
+## Members only (Authentication)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+Project based on authenticating users on a given web page.
```
Comment (javitocor) [inline README.md]: Be more descriptive, this is the first line everybody sees when it gets into your repo

File: README.md [hunk 2]
```diff
@@ -1,24 +1,67 @@
...
 
-Things you may want to cover:
+## Built with
 
-* Ruby version
+- Ruby
+- Ruby on Rails 
+- VSCode 
+- Linters
```
Comment (javitocor) [inline README.md:19]: Add the linters you are using

File: README.md [hunk 3]
```diff
@@ -1,24 +1,67 @@
...
+# Authors
 
-* ...
+## Author1
+👤 **Eduardo Baeza**
+
+- GitHub: [@githubhandle](https://github.com/edxco/)
+- Twitter: [@twitterhandle](https://twitter.com/lalo_nbc/)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/eduardo-n-baeza/)
+
```
Comment (javitocor) [inline README.md]: Add your usernames for every field

File: README.md [hunk 4]
```diff
@@ -1,24 +1,67 @@
...
+Forking the project.
+Cloning the project to your local machine.
+cd into the project directory.
+Run git checkout -b your-branch-name.
+Make your contributions.
+Push your branch up to your forked repository.
+Open a Pull Request with a detailed description to the development branch of the original project for a review.
+
+## Show your support
```
Comment (javitocor) [inline README.md:84]: Add acknowledgements section, we must be thankful

**Review by javitocor — CHANGES_REQUESTED** [review]
# STATUS CHANGES REQUIRED 🛑 

Hello team!
A great effort in this project! 👍 
Looks pretty good so far! 🥇 
The design is cool! 🥳 
However, we still need to implement some changes to get it approved 💪 
Kindly check my comments below 👇  make the changes and resubmit your project for a new code review ⏩ 
Good luck! 😺 
## Mandatory changes
- Kindly make your development branch the default branch
- Commit more often and be more descriptive
- Fill up the about section in your repo, it is good to give the maximum information possible, do not forget to add tags as well
- In your post controller, follow the instructions in the odin project, kindly create your own helper
    it uses `before_action` to restrict access to the `:new` and `:create` methods (uses ApplicationController helper method) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/authentication/before_action_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/authentication))
- Add the author in your posts show view, only for the signed-in users.

**Comment by edxco** [comment]
**ApplicationController helper method:**
Screenshot from 2020-11-25 11-29-37

**Showing author when user sign in**
Screenshot from 2020-11-25 11-31-19

**Author not showing when user log out**
Screenshot from 2020-11-25 11-31-30

**Review by javitocor — APPROVED** [review]
# STATUS APPROVED 🌟 🥇 

Congratulations! you have met all the requirements! 🥳 
Remember to fill up the completion form 📑  and you can move into the next project ⏩ 
Good luck! 😃

**Pull request merged**